### PR TITLE
Question bank UI changes

### PR DESCRIPTION
**Pull request by JagadishPujari**
1. Question bank action buttons and title alignment.
2. Select(checkbox) on question preview should load.
3. Limit question title.

Question bank UI issues - https://project-sunbird.atlassian.net/browse/SB-6445
The user can copy only v2 questions (https://project-sunbird.atlassian.net/browse/SB-6353)

**Scenarios**
* User can copy only v2 questions(show copy icon)
* User cant copy v1 questions(Those are created in ekstep platform without framework)

### CODE REVIEW

File: editor/questionbankcontroller.js [hunk 1]
```diff
@@ -357,6 +357,8 @@ angular.module('createquestionapp', [])
      *  @param {Object} selQuestion Selected question object
      */
     $scope.selectQuestion = function (selQuestion) {
+      //play preview
+      $scope.previewItem(selQuestion, true);
```
Comment (vinukumar-vs) [inline editor/questionbankcontroller.js:361]: This is for what?

File: editor/questionbankmodal.html [hunk 2]
```diff
@@ -74,26 +74,27 @@
           <div class="seven wide column qb-question-container" ng-show="!isQuestionSetConfig">
             <div class="row qb-question-list-container">
               <div class="ui large divided items">
-                <div class="item select list-items" ng-repeat="question in questions" ng-class="{quesSelector: $index === selectedQueIndex}">
-                  <div class="middle aligned content" ng-click="previewItem(question, true); showSelectedQue($index);">
-                    <div class="header">{{ question.title }}</div>
+                <div class="item select list-items ui grid" ng-repeat="question in questions" ng-class="{quesSelector: $index === selectedQueIndex}">
+                  <div class="middle aligned content ten wide column" ng-click="previewItem(question, true); showSelectedQue($index);">
+                    <div class="header">{{ question.title | limitTo:100}}</div>
```
Comment (vinukumar-vs) [inline editor/questionbankmodal.html]: Why it is limiting to 100 characters. As i told, 2nd line should have ellipse at the end.

File: editor/style.css [hunk 3]
```diff
@@ -222,7 +222,13 @@
     display: flex;
     align-items: center;
 }
-
+.action-btn-checkbox{
+    padding-top: 6px;
```
Comment (vinukumar-vs) [inline editor/style.css:235]: Why are you adding specific to only checkbox? What about copy & delete icon?

**Review by vinukumar-vs — CHANGES_REQUESTED** [review]

File: editor/questionbankcontroller.js (hunk 1 above)
Comment (JagadishPujari) [reply editor/questionbankcontroller.js:361]: Play preview clicking on edit question.

File: editor/style.css (hunk 3 above)
Comment (JagadishPujari) [reply editor/style.css:235]: Checkbox always have some padding so its moving top so

**Review by vinukumar-vs — APPROVED** [review]

**Pull request merged**